performance-storage-service/pss_project/api/models/rest/metrics/LatencyMetrics.py

```diff
@@ -0,0 +1,20 @@
+class LatencyMetrics(object):
+    def __init__(self, l_25=None, l_75=None, l_90=None, l_95=None, l_99=None, avg=None, median=None, max=None, min=None):
+        if l_25:
+            self.l_25 = l_25
+        if l_75:
+            self.l_75 = l_75
+        if l_90:
+            self.l_90 = l_90
+        if l_95:
+            self.l_95 = l_95 
+        if l_99:
+            self.l_99 = l_99 
+        if avg:
+            self.avg = avg
+        if median:
+            self.median = median
+        if max:
+            self.max = max
+        if min:
+            self.min = min
```

Comment:
I am a bit confused. 
- If users do not supply a parameter, its default value `None` will not be assigned to `self`, where the value will be `None` by default. There seems no different effect for having the `if` condition for the check compared to no `if` condition
- `if` condition in python is implicit. If the value is `0`, it will also give you false. Is this aligned with our intention for the checking?

Reply:
I might not quite understand what python does. What happens in this scenario:
```
latency = LatencyMetrics(max=34)
print(latency.l_99)
```
I was assuming that would throw an error as opposed to returning None. I wasn't sure which was better setting everything to `None` or leaving out the attributes. What are your thoughts?

Reply:
I think setting them to `None` or leaving them out works the same, as in both prints `"None"` upon the print statement.